Userland/head.cpp

```diff
@@ -1,25 +1,87 @@
+#include <errno.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <LibCore/CArgsParser.h>
+
+int head(String filename, bool print_filename, int line_count);
```

Comment:
`String` should be passed by const reference:
```suggestion
int head(const String& filename, bool print_filename, int line_count);
```